test-suite/generated-src/cpp/access_flags.hpp

```diff
@@ -52,5 +52,11 @@ struct hash<::testsuite::access_flags> {
         return std::hash<unsigned>()(static_cast<unsigned>(type));
     }
 };
+template <>
+class numeric_limits<::testsuite::access_flags> : public numeric_limits<unsigned> {
+    public:
+    static constexpr bool is_specialized = true;
+    static constexpr ::testsuite::access_flags max() noexcept { return static_cast<::testsuite::access_flags>(9); }
```

Comment:
I don't think this calculation is correct for flags.  A flag type like this may have 9 values, but its range is a full 9 bits.